deployments/mumbai/.chainId

```diff
@@ -0,0 +1 @@
+80001
```

Comment:
is this file naming convention a standard elsewhere or used somehow by the deploy plugin?

Reply:
yeah this is `hardhat-deploy`'s bookkeeping.